week-2/C-comparison-operators/exercise.js

```diff
@@ -8,14 +8,21 @@
 var studentCount = 16;
 var mentorCount = 9;
 var moreStudentsThanMentors; // finish this statement
+if (studentCount > 10) {
```

Comment:
Why compare `studentCount` to 10? There's a variable you're not using...

Reply:
@LucyMac will make correction, thanks